x-pack/filebeat/docs/inputs/input-aws-s3.asciidoc

```diff
@@ -212,8 +212,8 @@ The default is `10 MiB`.
 [float]
 ==== `max_number_of_messages`
 
-The maximum number of SQS messages that can be inflight at any time. Defaults
-to 5.
+The maximum number of SQS messages that can be inflight at any time, with a maximum of 10. Defaults
```

Comment:
Is this in response to some error observed while using the input?

If so then I think there may be an issue in the code. I intended to allow `max_number_of_messages` > 10 while only calling `ReceiveMessage` with a maximum value of 10. So the input would attempt to maximum the usages of the workers while splitting up the calls to `ReceiveMessage` into batches no larger than 10.